.gitlab-ci.yml

```diff
@@ -163,19 +162,23 @@ test_all_crystal:
 test_isolated:
   <<: *src_job_template
   stage: test
-  image: osrf/ros2:nightly
   only:
     - schedules
   dependencies:
     - build_isolated
   script:
+    - env  # For debugging
     - pip3 install mock
     - tar xf launchtest_artifacts.tar -C .
     - colcon test --packages-select apex_launchtest --pytest-args "-k not (test_flake8 or test_pep257)"
     - colcon test-result --all --verbose
     - tar xf launchtest_ros_artifacts.tar -C .
     - colcon test --packages-select apex_launchtest_ros --pytest-args "-k not (test_flake8 or test_pep257)"
     - colcon test-result --all --verbose
+  artifacts:
```

Comment:
This will help debug test failures.  The error message printed to the console is very sparse

Reply:
You could also add `--event-handler console_cohesion+`, but I think there is value in having the logs saved anyway